apify-docs-theme/src/theme/MDXComponents/Code.js

```diff
@@ -0,0 +1,42 @@
+import React, { isValidElement } from 'react';
```

Comment:
had to eject this completely, docusaurus now forbids wrapping of this component for some reason